scripts/deploy_claimManager.ts

```diff
@@ -0,0 +1,13 @@
+//deploy proxy for ClaimManager
+import { deployProxy, upgradeProxy } from "@openzeppelin/truffle-upgrades";
+import { ClaimManager } from "../ethers/ClaimManager";
+import { contract } from "truffle";
+import { deployer } from "@truffle/deployer";
+
+const claimManager = contract(ClaimManager);
+export async function deployClaimManager(erc1056: string, ensRegistry: string) {
```

Comment:
I'm confused about this script. All of the other scripts in this directory have the actual Volta addresses, why doesn't this one?
How does this script relate to 2_claimManager_migration.js?

Reply:
I created this deploy script to be used anywhere so thought of generalising it. The caller of the deploy method could simply pass the addresses( may or may not be the already deployed contract address).

Reply:
Ok, how does this script relate to `2_claimManager_migration.js` then? Is `2_claimManager_migration.js` what we would actually use to deploy the new IdentityManager to Volta? Or would we use this script?

Reply:
You made a valid point, we could use either of these. I wasn't sure if the `truffle` tests would run without migrations file so left it as it is. So, In a way the deploy script could be used just for development and testing without directly using truffle to deploy via migrations. I think we can optimise the implementation.